testing-and-tdd/testing-and-tdd.md

```diff
@@ -107,12 +107,12 @@ But what if your `class` uses another `class`? And what if your `function` calls
   - Since it's all isolated it shouldn't suffer from weird effects.
   - You can even mock the system clock to remove time dependencies! :-O
 - Realism: **low**.
-  - It's been said unit tests test that the code does what the developer intended, not what the user necessarily wants.
+  - It's been said that a unit test, tests that the code does what the developer intended, not what the user necessarily wants.
```

Comment:
I'm not sure about this one.  What about:

```suggestion
  - It's been said that a 'unit test' tests that the code does what the developer intended - not necessarily what the user wants.
```

Reply:
@LeiaQuesada, I second @alodahl's suggested change. What do you think?

Reply:
Absolutely! That makes sense!